samza-test/src/main/java/org/apache/samza/test/framework/TestRunner.java

```diff
@@ -384,4 +396,15 @@ private TaskFactory createTaskFactory() {
           new EndOfStreamMessage(null)));
       });
   }
+
+  /**
+   * Only adds a config from {@code config} to samza job {@code configs} if they dont exist in it.
```

Comment:
This java doc comment is confusing. Might be good to rename  to the following(or something better):
```
Adds new configuration from {@param config} to samza application configuration {@code configs}
```